src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -373,10 +465,19 @@ ApiGlobe.prototype.setRealisticLightingOn = function setRealisticLightingOn(valu
  * @params {visible} boolean.
  */
 
-ApiGlobe.prototype.setLayerVisibility = function setLayerVisibility(id, visible) {
-    this.scene.getMap().setLayerVisibility(id, visible);
-    this.update();
-    eventLayerChangedVisible.layerId = id;
+ApiGlobe.prototype.setLayerVisibility = function setLayerVisibility(layer, visible) {
```

Comment:
We agreed that: 
- this whole file is going away (see https://github.com/iTowns/itowns2/issues/272#issuecomment-288035531)
- itowns documentation needs a complete overhaul
So I don't think it's worth nit picking on doc for this particular PR.